File: fundamental-data-structures/string/questions/reversestring/reversestring_test.go
package reversestring

import (
	"fmt"
	"testing"
)

func Test(t *testing.T) {

	case1 := []byte("hello")
	fmt.Println(case1)
	ReverseString(case1)

	case2 := []byte("Hannah")
	fmt.Println(case2)
	ReverseString(case2)

	fmt.Println(case1)
	ReverseString1(case1)

	fmt.Println(case2)
	ReverseString1(case2)

	fmt.Println()
	fmt.Println(case1)
	ReverseString2(case1)

	fmt.Println(case2)
	ReverseString2(case2)
}

File: fundamental-data-structures/string/questions/reversestring/ReverseStringTest.kt
package string.questions.reversestring

fun main() {
    val case1 = charArrayOf('h','e','l','l','o')
    case1.forEach { print("$it,") }
    println()
    ReverseString.reverseString(case1)
    case1.forEach { print("$it,") }

    val case2 = charArrayOf('H','a','n','n','a','h')
    println()
    case2.forEach { print("$it,") }
    println()
    ReverseString.reverseString(case2)
    case2.forEach { print("$it,") }
}